refinery/core/tests.py

```diff
@@ -788,6 +788,18 @@ def get_credentials(self):
             password=self.password
         )
 
+    def test_xml_format_ignored(self):
+        response = self.api_client.get(
+            '/api/v1/analysis/',
```

Comment:
`/api/v1/`? otherwise this is `test_xml_format_ignored_in_analysis_api`

Reply:
Sorry, I'm missing your point... do you just want the method renamed?

Reply:
I was joking about renaming the method. The API URL should be `/api/v1/`, otherwise this creates a tight coupling with the analysis endpoint. In principle, all the endpoints should be tested but testing just the root of the API should suffice for our purposes.